pkg/controller/compliance/compliance_controller.go

```diff
@@ -328,14 +354,19 @@ func (r *ReconcileCompliance) Reconcile(ctx context.Context, request reconcile.R
 		return reconcile.Result{}, err
 	}
 
-	certificateManager, err := certificatemanager.Create(r.client, network, r.clusterDomain, common.OperatorNamespace())
+	var opts []certificatemanager.Option
+	if tenant != nil {
+		opts = append(opts, certificatemanager.WithTenant(tenant))
```

Comment:
The `WithTenant` option actually handles a nil tenant behind the scenes, so no need for the `tenant != nil` check here.

Reply:
Would be nice to include a `WithLogger` option here to pass the reqLogger context logger.